src/state/index.tsx

```diff
@@ -43,11 +43,14 @@ export const StateContext = createContext<StateContextType>(null!);
 export default function AppStateProvider(props: React.PropsWithChildren<{}>) {
   const [error, setError] = useState<TwilioError | null>(null);
   const [isFetching, setIsFetching] = useState(false);
-  const [isGridViewActive, setIsGridViewActive] = useLocalStorageState('grid-view-active-key', true);
+  const [isGalleryViewActive, setIsGalleryViewActive] = useLocalStorageState('gallery-view-active-key', true);
   const [activeSinkId, setActiveSinkId] = useActiveSinkId();
   const [settings, dispatchSetting] = useReducer(settingsReducer, initialSettings);
   const [roomType, setRoomType] = useState<RoomType>();
-  const [maxGridParticipants, setMaxGridParticipants] = useLocalStorageState('max-grid-participants-key', 9);
+  const [maxGalleryViewParticipants, setMaxGalleryViewParticipants] = useLocalStorageState(
+    'max-gallery-participants-key',
```

Comment:
Nice, changing the name of the key here will actually reset everyone's saved preference, which is good since most people probably have 4 or 9 saved. If we didn't change the key, then some people will report that their Max Gallery Participants setting appears to be completely blank, because the saved option doesn't match up with an entry in the drop down.